src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -1325,8 +1325,14 @@ void TwitchMessageBuilder::appendTwitchBadges()
             auto badgeInfoIt = badgeInfos.find(badge.key_);
             if (badgeInfoIt != badgeInfos.end())
             {
+                const auto &subTier =
```

Comment:
Could you add a comment describing how the sub tier is calculated with some examples?

Reply:
Added in b74c7de. Hope it's clear enough ![](https://cdn.betterttv.net/emote/5e17b9a6b974112104806a1c/1x)